### PR TITLE
[ID] Update `Beatmap/Overall difficulty`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
SKIP_OUTDATED_CHECK 

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Nivalyx** [comment]
oh you'll need to add `SKIP_OUTDATED_CHECK` to your commit post btw since you changed something in the original English page as well :

image

(or else the pr will always fail the remark check orz)

File: wiki/Beatmap/Overall_difficulty/en.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ tags:
 
 Higher OD values mean shorter time windows to clear [hit objects](/wiki/Hit_object), both in general and in terms of getting high [score](/wiki/Gameplay/Score) values. The maximum allowed hit error for each hit object in osu! and osu!mania, centred on the hit object's correct time, is defined by the tables below.
 
-Note that in the stable version of osu!, hit windows in osu! and osu!taiko can effectively be up to 0.5 ms shorter on both sides than what the formulae suggest, and in osu!mania they can be up to 0.5 ms longer on both sides. This is because in osu! and osu!taiko, a hit is considered inside a hit window if `hit error < round(hit window)`, while in osu!mania it is considered inside if `hit error <= round(hit window)`.
+Note that in the stable version of osu!, hit windows in osu! and osu!taiko can effectively be up to 0.5 ms shorter on both sides than what the formula suggest, and in osu!mania they can be up to 0.5 ms longer on both sides. This is because in osu! and osu!taiko, a hit is considered inside a hit window if `hit error < round(hit window)`, while in osu!mania it is considered inside if `hit error <= round(hit window)`.
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/en.md:19]: `formulae` is actually correct tho, as it's an alternative spelling of `formulas` (http://writingexplained.org/formulas-or-formulae-correct)

...although yeah i looked around the wiki and other articles such as http://osu.ppy.sh/wiki/en/Beatmap/Approach_rate and http://osu.ppy.sh/wiki/en/Client/Beatmap_editor/Song_Setup went on to use `formulas` instead

File: wiki/Beatmap/Overall_difficulty/id.md [hunk 2]
```diff
@@ -16,21 +14,47 @@ outdated_since: 25fa617b062ef73cd5eda1246697cc0dfdc1862b
 
 ## Timing
 
-Semakin tinggi nilai OD, semakin pendek jeda waktu untuk menekan [hit object](/wiki/Hit_object), baik secara umum maupun dalam mendapatkan [skor](/wiki/Gameplay/Score) yang tinggi. Besar dari hit window, berpusat pada ketepatan waktu hit object, dijelaskan dengan tabel berikut:
+Semakin tinggi nilai OD, semakin pendek jeda waktu untuk menekan [hit object](/wiki/Hit_object), baik secara umum maupun dalam mendapatkan [skor](/wiki/Gameplay/Score) yang tinggi. Waktu maksimum hit error untuk setiap hit object pada mode permainan osu! dan osu!mania berpusat pada ketepatan waktu hit object, dijelaskan dengan tabel di bawah.
 
-| Skor | Hit window |
+Perlu dicatat pada osu! versi stable, hit window mode permainan osu! dan osu!taiko dapat lebih pendek 0,5 ms daripada hasil rumus perhitungan di kedua sisi (sebelum dan sesudah waktu hit), namun hal sebaliknya dapat terjadi pada osu!mania. Hal tersebut disebabkan oleh bagaimana sebuah hit dinilai, pada mode permainan osu! dan osu!taiko sebuah hit berada pada hit window jika `hit error < pembulatan(hit window)`, namun pada mode permainan osu!mania sebuah hit berada pada hit window jika `hit error <= pembulatan(hit window)`.
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/id.md:19]:
```suggestion
Perlu dicatat pada osu! versi stable, hit window mode permainan osu! dan osu!taiko dapat lebih pendek 0,5 ms daripada hasil rumus perhitungan baik sebelum maupun sesudah waktu hit. Meskipun demikina, hal yang sebaliknya justru dapat terjadi pada mode osu!mania. Hal ini dikarenakan oleh mekanisme perhitungan waktu hit itu sendiri, di mana pada mode permainan osu! dan osu!taiko sebuah hit berada pada hit window apabila `hit error < pembulatan(hit window)`, namun pada mode permainan osu!mania sebuah hit berada pada hit window apabila `hit error <= pembulatan(hit window)`.
```

File: wiki/Beatmap/Overall_difficulty/en.md (hunk 1 above)
Comment (ZeroPyrozen) [reply wiki/Beatmap/Overall_difficulty/en.md:19]: hmm TIL, should we change it to `formulas` instead?

File: wiki/Beatmap/Overall_difficulty/id.md [hunk 3]
```diff
@@ -16,21 +14,47 @@ outdated_since: 25fa617b062ef73cd5eda1246697cc0dfdc1862b
...
+
+Sebagai contoh, hit window sebuah Great pada osu!taiko dengan OD 5 adalah ±34,5 ms, alih-alih ±35 ms seperti hasil rumus pada tabel. Pada mode permainan osu!mania, hit window sebuah MAX adalah ±16,5 ms, bukan ±16 ms seperti hasil rumus pada tabel.
+
+Nilai hit window untuk penilaian-penilaian dapat dilihat dengan mengarahkan kursor di atas [informasi beatmap pada menu pemilihan lagu](/wiki/Client/Interface#informasi-beatmap), yang akan selalu memberikan nilai yang tepat.
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/id.md:23]:
```suggestion
Nilai hit window untuk berbagai waktu penilaian yang ada dapat dilihat dengan mengarahkan kursor di atas [informasi beatmap pada menu pemilihan lagu](/wiki/Client/Interface#informasi-beatmap), yang akan selalu memberikan nilai yang tepat.
```

File: wiki/Beatmap/Overall_difficulty/id.md [hunk 4]
```diff
@@ -16,21 +14,47 @@ outdated_since: 25fa617b062ef73cd5eda1246697cc0dfdc1862b
...
-| 100 | `280ms - 16ms * OD` |
-| 300 | `160ms - 12ms * OD` |
+| MAX | `16` |
+| 300 | `64 - 3 * OD` |
+| 200 | `97 - 3 * OD` |
+| 100 | `127 - 3 * OD` |
+| 50 | `188 - 3 * OD` |
 
-Jika pemain menekan hit object diluar dari 50 hit window, itu akan dihitung sebagai miss. Pada kasus hit window antara dua objek yang bertumpuk, objek kedua tidak akan bisa ditekan hingga objek pertama hilang karena mekanik [notelock](/wiki/Gameplay/Judgement/Notelock).
+Jika pemain menekan hit object diluar dari hit window 50, pemain akan mendapatkan miss. Pada kasus hit window antara dua objek yang bertumpuk, objek kedua tidak akan bisa ditekan hingga objek pertama hilang karena mekanik [notelock](/wiki/Gameplay/Judgement/Notelock).
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/id.md:51]:
```suggestion
Jika pemain menekan hit object di luar dari hit window 50, pemain akan mendapatkan miss. Apabila terdapat dua objek yang hit window-nya saling bertumpuk, objek kedua tidak akan dapat ditekan hingga objek pertama hilang akibat mekanika [notelock](/wiki/Gameplay/Judgement/Notelock).
```

File: wiki/Beatmap/Overall_difficulty/id.md [hunk 5]
```diff
@@ -16,21 +14,47 @@ outdated_since: 25fa617b062ef73cd5eda1246697cc0dfdc1862b
...
 
 ## Slider dan spinner
 
-Di [osu!](/wiki/Game_mode/osu!), slider akan memberikan 300 selama mereka ditekan dalam 50 hit window. Ini terkadang disebut sebagai *slideracc* dan dihapus di dalam [ScoreV2](/wiki/Game_modifier/ScoreV2).
+Pada mode permainan [osu!](/wiki/Game_mode/osu!), slider akan memberikan skor 300 selama mereka ditekan dalam hit window 50. Ini terkadang disebut sebagai *slideracc* dan dihapus di dalam [ScoreV2](/wiki/Game_modifier/ScoreV2).
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/id.md:55]:
```suggestion
Pada mode permainan [osu!](/wiki/Game_mode/osu!), slider akan memberikan skor 300 selama mereka ditekan dalam hit window 50. Hal ini terkadang disebut dengan istilah *slideracc* yang telah dihapus di dalam [ScoreV2](/wiki/Game_modifier/ScoreV2).
```

File: wiki/Beatmap/Overall_difficulty/id.md [hunk 6]
```diff
@@ -16,21 +14,47 @@ outdated_since: 25fa617b062ef73cd5eda1246697cc0dfdc1862b
...
 
-Overall difficulty juga berefek pada [spinner](/wiki/Hit_object/Spinner), di mana mereka harus memutar lebih untuk mengisi pengukur pada waktunya. Di [osu!taiko](/wiki/Game_mode/osu!taiko), denden juga membutuhkan hit yang lebih banyak untuk diselesaikan. Spin per detik sangat dibutuhkan untuk menyelesaikan spinner, yang dimana akan dijelaskan dengan rumus berikut:
+Overall difficulty juga berefek pada [spinner](/wiki/Hit_object/Spinner), di mana mereka harus diputar lebih banyak untuk mengisi pengukur pada waktunya. Pada mode permainan [osu!taiko](/wiki/Game_mode/osu!taiko), denden juga perlu ditekan lebih banyak untuk diselesaikan. Jumlah putaran per detik sangat dibutuhkan untuk menyelesaikan spinner, yang dijelaskan dengan rumus berikut:
```
Comment (Nivalyx) [inline wiki/Beatmap/Overall_difficulty/id.md:57]:
```suggestion
Overall difficulty juga berefek pada [spinner](/wiki/Hit_object/Spinner), di mana mereka harus diputar lebih banyak agar *spinner metre* yang ada dapat terisi pada waktunya. Pada mode permainan [osu!taiko](/wiki/Game_mode/osu!taiko), denden juga perlu ditekan lebih banyak untuk dapat diselesaikan. Di samping itu, jumlah putaran per detik yang dibutuhkan untuk menyelesaikan spinner juga akan terdampak sesuai dengan rumus berikut:
```

**Review by Nivalyx — APPROVED** [review]
checked, looks fine for the `id.md` side for me

**Review by TPGPL — APPROVED** [review]

**Pull request merged**